components/Header/index.js

```diff
@@ -8,5 +8,39 @@
 //    <span class="temp">98°</span>
 //  </div >
 // And add it to the DOM in the .headerContainer component
+const head = document.querySelector('.header-container');
 
-function Header() {}
+head.appendChild(Header());
+
+function Header() {
+
+      // date if we wanted the real date
```

Comment:
I love that you added a portion of code to add the current date if we wanted too!